pkg/transforms/policyreport.go

```diff
@@ -5,35 +5,36 @@ package transforms
 
 import (
 	"strings"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // PolicyReport report
 type PolicyReport struct {
-	metav1.TypeMeta `json:",inline"`
+	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
-	Results []ReportResults `json:"results"`
+	Results           []ReportResults `json:"results"`
 }
 
 // ReportResults rule violation results
 type ReportResults struct {
-	Policy string `json:"policy"`
-	Message string `json:"message"`
-	Category string `json:"category"`
-	Status string `json:"status"`
-	Data ReportData `json:"data"`
+	Policy      string     `json:"policy"`
+	Description string     `json:"message"`
+	Category    string     `json:"category"`
+	Result      string     `json:"result"`
+	Properties  ReportData `json:"properties"`
```

Comment:
Should we add timestamp here as well?

Reply:
Not sure we want Timestamp in search , until we need it we dont want to cache in redis .